### ISSUE TITLE
Replacing MG Sounds

### ISSUE THREAD

**Issue by Papaimpossible** [GaijinEntertainment/fmod_studio_warthunder_for_modders#233]
Im trying to change the m1919 interior as well as exterior sounds. Sounds good in events but when I build and test, the game has stock mg sounds somehow which is very confusing because I thought I replaced all close far and inside wavs as well as a majority of the ones in grains as well. 

So far I've been able to fully replace m4a1 engine sound as well as 75mm m3 interior and exterior with using tank and tank_open respectively so I thought if I try that on the mgs it would work as well. it didnt. Ended up tweaking with the master bus for tank and tank_open and ended up enabling all mg and following trees correlating to it yet nothing as well as setting in and out bus to auto.

video showing what im talking about
https://www.youtube.com/watch?v=T5gu7-X7sNk&ab_channel=HigerHunter

**Comment by NukprachTH** [comment]
I have this same problem too, I still don't know how to solve this.

**Comment by Papaimpossible** [comment]
Update: 
I just remade M2 50 Cal sounds right now and tried everything I said above again and it seems like the game doesn't register the sound change if that's even possible. Default sounds still play yet all of the wavs that involve the firing sequence was replaced with my custom wavs but there still is no change.

Might just be a mg thing? I don't have any problems with the cannons or engines

**Comment by IWSM** [comment]
You need use a good Fmod studio version guys

**Comment by Papaimpossible** [comment]
> You need use a good Fmod studio version guys

I'm using 2.01.06

**Comment by IWSM** [comment]
You use a asset .bank with tank_weapons.bank? Your wav in on asset.bank

**Comment by Papaimpossible** [comment]
yes i replaced the wavs in assets and when i build it they're seperate. drag all the files into the mods folder in sounds. loops are on in config, turned it off and it made no difference

**Comment by IWSM** [comment]
Not (mods) folders, correct name is (mod) folder dude

**Comment by Papaimpossible** [comment]
it is mod in file explorer

**Comment by IWSM** [comment]
Show me your mod folder pls

**Comment by Papaimpossible** [comment]
![Screenshot (746)](https://github.com/GaijinEntertainment/fmod_studio_warthunder_for_modders/assets/143297155/6781fb39-c733-43a3-b845-82aecc57efca)

**Comment by IWSM** [comment]
Your are add line on config.blk?

**Comment by Papaimpossible** [comment]
![Screenshot (747)](https://github.com/GaijinEntertainment/fmod_studio_warthunder_for_modders/assets/143297155/8b83058d-b7a2-43c7-ade7-161885cbcb75)

everythings on in config i even tried useLoopsForGMGunSound but it ended up messing up the audio

**Comment by IWSM** [comment]
you don't need useloop. there's something you're doing wrong somewhere.

**Comment by Papaimpossible** [comment]
i turned loops off and nothing changed

**Comment by IWSM** [comment]
I told you, you weren't to use it's no use.
